README.md

```diff
@@ -6,37 +6,37 @@
 [![Stack Overflow](https://img.shields.io/badge/stackoverflow-grakn-796de3.svg)](https://stackoverflow.com/questions/tagged/grakn)
 [![Stack Overflow](https://img.shields.io/badge/stackoverflow-graql-3dce8c.svg)](https://stackoverflow.com/questions/tagged/graql)
 
-Grakn is a distributed knowledge graph: a logical database to organise large and complex networks of data as one body of knowledge.
+Grakn is a distributed knowledge graph: a logical database to organize large and complex networks of data as one body of knowledge.
```

Comment:
We would spell it "organise" on this side of the ocean :) Can we revert back the English spelling?